pages/jdl.md

```diff
@@ -291,6 +291,7 @@ entity C
 skipClient for A
 skipServer for B
 angularSuffix * with mySuperEntities
+filter C
```

Comment:
I didn't know about this one, @MathieuAA can you confirm?

Reply:
Sorry didn't see anything as I came back from vacations and didn't have the time to use my computer since then... Yes it's correct.